lib/acts_as_paranoid/core.rb

```diff
@@ -36,7 +36,12 @@ def delete_all!(conditions = nil)
       end
 
       def delete_all(conditions = nil)
-        where(conditions).update_all(["#{paranoid_configuration[:column]} = ?", delete_now_value])
+        ar = where(conditions)
+        if paranoid_configuration[:touch]
+          ar.update_all(["#{paranoid_configuration[:column]} = ?, updated_at = ?", delete_now_value, Time.now])
```

Comment:
In the AR world I think it's safe to assume that the `updated_at` field is a timestamp.